EIPS/eip-2098.md

```diff
@@ -0,0 +1,212 @@
+---
+eip: 2098
+title: Compact Signature Representation
+status: Draft
+type: Informational
+author: Richard Moore <@ricmoo>, Nick Johnson <nick@ethereum.org>
+discussions-to: https://github.com/ethereum/EIPs/issues/2440
+created: 2019-03-14
+---
+
+## Simply Summary
+
+This proposal describes a compact representation of an Ethereum Signature.
+
+
+## Abstract
+
+The secp256k1 curve permits the computation of the public key of signed
+digest when coupled with a signature, which is used implicitly to
+establish the origin of a transacion from an Externally Owned Account
+as well as on-chain in EVM contracts for example, in meta-transactions and
+multi-sig contracts.
+
+Currently signatures require 65 bytes to represent, which when aligned
+to 256-bit words, requires 96 bytes (with 31 zero bytes injected). With
+compact signatures, this can be reduced to 64 bytes, which remains 64
+bytes when word-aligned.
+
+
+## Motivation
+
+The motivations for a compact representation are to simplify handling
+transactions in client code, reduce gas costs and reduce transaction sizes.
+
+
+## Specification
+
+A secp256k1 signature is made up of 3 parameters, `r`, `s` and `v`. The `r`
+represents the `x` component on the curve (from which the `y` can be
+computed), and the `s` represents the challenge solution for signing by a
+private key. Due to the symmetric nature of an elliptic curve, a `v` is
+required, which indicates which of the 2 possible solutions was intended,
+by indicating its parity (odd-ness), 
+
+Two key observation are required to create a compact representation.
+
+First, the `v` parameter is always either 0 or 1 (canonically the values used
+have been 27 and 28, as these values didn't collide with other binary prefixes
+used in Bitcoin.)
+
+Second, the top bit of the `s` parameters is **always** 0, due to the use of
+canonical signatures which flip the solution parity to prevent negative values,
+which was introduced as [a contraint in homestead](http://eips.ethereum.org/EIPS/eip-2).
+
+So, we can use the top bit in the `s` parameter to store `v`.
+
```

Comment:
I'd suggest adding representation of the serialised 64-bytes here (similar to how many RFCs have it).

Perhaps:

```suggestion
`[256-bit r value][1-bit v value][255-bit s value]`
```

Reply:
Added a few more words around it, but added this. :)